static/app/components/events/eventEntry.tsx

```diff
@@ -158,22 +155,10 @@ function EventEntry({
         group?.issueCategory === IssueCategory.PERFORMANCE &&
         organization?.features?.includes('performance-issues')
```

Comment:
Small nit, but I think this condition would be even better as 
```suggestion
        event?.type === EventOrGroupType.TRANSACTION &&
        organization?.features?.includes('performance-issues')
```

Because 

1. `<SpanEvidenceSection` doesn't even use the group, so it's strange that it has to check the group!
2. TypeScript's type narrowing will kick in, and you won't need to do `as EventTransaction` below

Reply:
@gggritso I was checking if it's a Performance issue here because the block of code under `case EntryType.SPANS:` was previously being used to do something else, and I'm unsure whether this might mess that up. I have a feeling it's no longer in use though, so this was just for insurance :)

I'm gonna verify and confirm that it's unused before making this change here